### DISCUSSION TITLE
Increase logging with verb command

### DISCUSSION THREAD

**Discussion by noaho** [schwabe/ics-openvpn#1368] [Q&A]
Hi,

I'm probably missing something obvious, but I'm trying to change to verb 3 so I can see what cipher is negotiated to troubleshoot another problem, however I can't see it as an option and if I add it to custom options it just gets added twice and thus ignored?

I'm on OpenVPN for Android 0.7.22

Cheers!

**Comment by schwabe** [comment]
The app already runs with `verb 4` by default. You can use the slider in the log window to change the shown verbosity
